### ISSUE TITLE
Jicofo's reservation.rest.BASE_URL cannot be specified as ENV

### ISSUE THREAD

**Issue by kheir95** [jitsi/docker-jitsi-meet#175]
Currently it's not possible to set the Base URL of Jicofo's reservation system from the `.env` file.
Instead `org.jitsi.impl.reservation.rest.BASE_URL=http://reservation.example.com` must be added manually to `.jitsi-meet-cfg/jicofo/sip-communicator.properties`, which get's resetted sometimes.
There is definitely some workarounds, but it'd be most elegant, if this feature was implemented, so that adding the URL to the `.env` file would to the trick.

**Comment by saghul** [comment]
Sounds good. A PR adding it would be most welcome.

**Comment by kheir95** [comment]
Sadly I don't have enough docker knowledge to complete this task fast, also my supervisor says a workaround would work for us, so I should do this at my free time.
It'll be really great if someone else, who have a better understanding to the structure could have look at it.

**Comment by saghul** [comment]
Allow me to rant a little.

This is why we can't have nice things. You supervisor is happy with whatever hack, disregarding the time they saved because this project exists in the first place.

The change required to enabe this functionality takes 10 minutes to do, if you have experience, I'd say a few hours if you don't know the setup well.

Good luck, just so you know, your supervisor is a bad one.

**Comment by kheir95** [comment]
I can fully understand your frustration and this is indeed a problem.
But it really isn't the case here. We tried for quite some time to find a way to add this feature, the structure was really not clear to do so fast and it seemed like it will take more than few hours. This project has already exceeded it's planned time and my supervisor had to make this decision.
We already opened an issue about a bug before and provided the required solution.
We are really grateful for the work you guys are doing with this project but sometimes cost must be taken into consideration too.
However I took some private time and figured this out, it appears to be simple once you get the structure.
